src/main/java/com/myjeeva/digitalocean/DigitalOcean.java

```diff
@@ -2051,4 +2055,49 @@ Delete removeDropletsFromFirewall(String firewallId, List<Integer> dropletIds)
   Firewalls getAvailableFirewalls(Integer pageNo, Integer perPage)
       throws DigitalOceanException, RequestUnsuccessfulException;
 
+  /**
+   * Method allows you to create a new project.
+   *
+   * @param project the project
+   * @return {@link Project}
+   * @throws DigitalOceanException if request had interruption [
+   *         <code>HTTP status code &gt;= 400 &amp;&amp; &lt; 510</code>]
+   * @throws RequestUnsuccessfulException if any RESTful request unsuccessful from wrapper method
+   */
+  Project createProject(Project project)
+      throws DigitalOceanException, RequestUnsuccessfulException;
+
+  /**
+   * Method allows you to get all the projects.
+   *
+   * @return {@link Projects}
+   * @throws DigitalOceanException if request had interruption [
+   *         <code>HTTP status code &gt;= 400 &amp;&amp; &lt; 510</code>]
+   * @throws RequestUnsuccessfulException if any RESTful request unsuccessful from wrapper method
+   */
+  Projects getAllProjects()
```

Comment:
@TrikThom Can you please update the method to `getAvailableProjects()` like other methods?